code-review-trainer-app/package-lock.json

```diff
@@ -0,0 +1,3758 @@
+{
```

Comment:
This file should not have been added. We should have yarn.lock and not package-lock.json